assets/main/world/environment/path-00-sheet.png.import

```diff
@@ -0,0 +1,34 @@
+[remap]
```

Comment:
This asset could have a better name. Maybe 'path-stone-sheet' after the path type.

Reply:
Renamed to `path-stone-sheet.png`